.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: objective-c
+cache: cocoapods
```

Comment:
I assume this is for `Pods` directory generated by `pod install` and will have no effect for `~/.cocoapods` you may be expecting.